weather_mv/loader_pipeline/pipeline.py

```diff
@@ -276,19 +278,27 @@ def run(argv: t.List[str], save_main_session: bool = True):
         prog='weather-mv',
         description='Weather Mover loads weather data from cloud storage into Google BigQuery.'
     )
+    parser.add_argument('-i', '--uris', type=str, required=True,
+                        help="URI prefix matching input netcdf objects, e.g. 'gs://ecmwf/era5/era5-2015-'.")
+    parser.add_argument('-o', '--output_table', type=str, required=True,
+                        help="Full name of destination BigQuery table (<project>.<dataset>.<table>). Table will be "
+                             "created if it doesn't exist.")
+    parser.add_argument('-t', '--temp_location', type=str, required=True,
+                        help='Cloud Storage path for temporary files, used by the BigQuery writer. Must be a valid '
+                             'Cloud Storage URL, e.g. beginning with gs://')
```

Comment:
I moved these arguments up since they are all required. I also changed the type of string characters (`"` vs `'`) for some of these.